util/pace/util/grid/generation.py

```diff
@@ -1529,6 +1560,64 @@ def rdyc(self) -> util.Quantity:
             gt4py_backend=self.dyc.gt4py_backend,
         )
 
+    def _init_cartesian(self):
+
+        domain_rad = PI / 16.0
+        lat_rad = self._deglat * PI / 180.0
+        lon_rad = 0.0
+
+        self._dx, self._dy = self._compute_dxdy_cartesian()
+        self._dx_agrid, self._dy_agrid = self._compute_dxdy_agrid_cartesian()
+        self._dx_center, self._dy_center = self._compute_dxdy_center_cartesian()
+        self._area = self._compute_area_cartesian()
+        self._area_c = self._compute_area_c_cartesian()
```

Comment:
Similar to the dx/dy computation above, these areas are also the same so you could have one `_set_area_cartesian` method that can do either/both of these instead of two separate methods

Reply:
Same as above: areas sit at different locations, but could still reduce code duplication.